.ci/macos-steps.yaml

```diff
@@ -16,8 +16,7 @@ steps:
     unset BOOST_ROOT
     pip install cython numpy pandas zipp
     brew update
-    brew install openblas armadillo boost hdf5@1.10
-    brew switch hdf5 1.10.6
+    brew install openblas armadillo@8.400 boost
```

Comment:
Hi @Yashwants19, we can't specify armadillo versions like this. I highly suggest testing brew commands locally. Thanks for working on this though. 👍

Reply:
If you can test this locally please try to unlink hdf5 and then install specific version of hdf5.
Thanks

Reply:
Will try it. Thanks.

Reply:
Hmm, This doesn't work either.